emcc.py

```diff
@@ -2213,6 +2219,18 @@ def check_memory_setting(setting):
 
     if settings.LINKABLE:
       exit_with_error('ASan does not support dynamic linking')
+  elif 'leak' in sanitize:
+    settings.USE_LSAN = 1
+    default_setting('EXIT_RUNTIME', 1)
+
+    if settings.LINKABLE:
+      exit_with_error('LSan does not support dynamic linking')
+
+  if sanitizer_mem:
+    # Increase the size of the initial memory according to how much memory
+    # we think the sanitizers will use.
+    logger.debug(f'adding {sanitizer_mem} bytes of memory for sanitizer support')
+    settings.INITIAL_MEMORY += sanitizer_mem
```

Comment:
same as above - this looks like a dupe of lines 2199-2212. I found that by searching for `settings.INITIAL_MEMORY += sanitizer_mem` which now appears twice in the code.